packages/validator/src/spec/PwaValidatorSpec.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import {PwaValidator} from '../lib/PwaValidator';
+import {PsiResult} from '../lib/psi';
+
+function mockPsiResult(performanceScore: number, pwaScore: number): PsiResult {
+  return {
+    lighthouseResult: {
```

Comment:
Is there any use in having `PsiResult` contain a bunch of fields that we never use?

Reply:
PsiResult is a mapping of the response from the PageSpeed Insights API (which is quite verbose).

Reply:
Yeah, but we don't need 90% of the fields right? We can just get rid of the ones we don't use and it won't (I think?) cause any errors.

It would make the code easier to understand and more robust - we don't need to worry about updating something when some unrelated part of the API changes.

Reply:
cleaned up.